cloudbuild.yaml

```diff
@@ -82,6 +82,12 @@ steps:
     - CLOUDBUILD=1
   waitFor: ['init']
 
+- name: 'gcr.io/oss-vdb/ci'
+  id: 'sourcerepo-sync-tests'
+  dir: 'tools/sourcerepo-sync'
+  args: ['bash', '-ex', 'run_tests.sh']
+  waitFor: ['init']
```

Comment:
@jess-lowe @another-rex I think this has a race condition when running `pipenv sync` with `lib-tests` (since they use the same environment)

It might need to be changed to
```yaml
waitFor: ['init', 'lib-tests']
```
(and maybe move this below lib-tests & add a comment  so the relation is easier to see)